### PR TITLE
Release notes 3.0

### PR DESCRIPTION
**Pull request by jcalcaben**
## Description

Update the CHANGELOG file with release notes for 3.0.0.

## Related Issue

An action item of #1239 

## Verification Steps
1. Open CHANGELOG.md
2. Verify 3.0.0 release notes content exists and covers all the work done for 3.0.0.

## Checklist:

- [x] I have updated the documentation accordingly, if necessary.
- [ ] I have added tests to cover my changes, if necessary.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,185 +1,172 @@
...
-[#916]: https://github.com/magento-research/pwa-studio/issues/916
-[#830]: https://github.com/magento-research/pwa-studio/issues/83o
-[#824]: https://github.com/magento-research/pwa-studio/issues/824
-[nodejs]: https://magento-research.github.io/pwa-studio/technologies/upward/reference-implementation/
-[php]: https://github.com/magento-research/magento2-upward-connector
-[time to first meaningful paint]: https://developers.google.com/web/tools/lighthouse/audits/first-meaningful-paint
-[GitHub query result]: https://github.com/magento-research/pwa-studio/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Arelease%2F2.0+review%3Aapproved+NOT+fix+in%3Atitle+NOT+fixed+in%3Atitle+NOT+chore+in%3Atitle+NOT+test+in%3Atitle&utf8=%E2%9C%93
+| Change type | Description                                                                 | PR        |
+| ----------- | --------------------------------------------------------------------------- | --------- |
+| Feature     | Layered navigation (filters) modal component                                | [#797][]  |
```
[the remark this answers is not in the document]
Comment (jcalcaben) [reply CHANGELOG.md]: Resolved in: b61d690

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,185 +1,172 @@
...
--   [Theme vs Storefront][] ([#509][])
--   [Install Venia sample data][] ([#474][])
--   [Peregrine Routing][] ([#453][])
+### Summary of major changes
 
-#### Updated topics
+- **Peregrine hooks:**
+    Custom [React Hooks][] that contain data and state management logic has been added to the Peregrine library.
+    These hooks allow developers to easily create functional components that focus on presentational logic.
+    This also allows for a more modular use of PWA logic and easier "restyling" of Venia (or any PWA Storefront built using PWA Studio).
```
[the remark this answers is not in the document]
Comment (jcalcaben) [reply CHANGELOG.md]: Resolved in: b61d690

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,185 +1,172 @@
...
 
--   [README][] for the `Magento_UpwardConnector` module, which contains Cloud deployment instructions.
+- [What's new in 3.0.0](#whats-new-in-300)
+- [Updating from 2.1.0](#updating-from-210)
 
-### Other documentation updates
+## What's new in 3.0.0
 
-#### New topics
+PWA Studio 3.0.0 contains improvements, new features, and some breaking changes.
```
[the remark this answers is not in the document]
Comment (jcalcaben) [reply CHANGELOG.md]: Breaking changes section added in b61d690

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,185 +1,183 @@
-# Release 2.0
+# Release 3.0
```
Comment (cherdman) [inline CHANGELOG.md]: Should be 3.0.0 per our versioning policy (semver)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,185 +1,183 @@
...
-* Bugfix for pagination persisting during Query loading state
-* Bugfix for Search autocomplete rendering loading component on clear
-
-## What's new in 2.0.0
-
-This is a brief development summary for the 2.0.0 release.
-
-For a list of relevant Pull Requests related to the 2.0 release, see this [GitHub query result][].
+**NOTE:**
+_This changelog only contains release notes for PWA Studio 3.0 and above._
```
Comment (cherdman) [inline CHANGELOG.md]: Should be PWA Studio 3.0.0, per our versioning policy (semver)

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,185 +1,183 @@
...
+| Change type | Description                           | PR        |
+| ----------- | ------------------------------------- | --------- |
+| Update      | Update `workbox-webpack-plugin` to v4 | [#1102][] |
+
+### Misc project changes
+
+| Change type    | Description                            | PR                                         |
+| -------------- | -------------------------------------- | ------------------------------------------ |
+| Update         | Update eslint configuration version    | [#1088][]                                  |
+| Infrastructure | CICD and devops fixes and improvements | [#1132][], [#1155][], [#1087][], [#1043][] |
```
Comment (cherdman) [inline CHANGELOG.md]: Should be 'DevOps' and not 'devops'. We could also consider changing CICD to CI/CD but that's more of a personal opinion.

**Pull request merged**